classes/Tools14.php

```diff
@@ -148,32 +148,6 @@ public static function deleteDirectory(string $dirname, bool $delete_self = true
         return false;
     }
 
-    /**
-     * Display an error according to an error code.
-     *
-     * @param string $string Error message
-     * @param bool $htmlentities By default at true for parsing error message with htmlentities
-     */
-    public static function displayError(string $string = 'Fatal error', bool $htmlentities = true): string
```

Comment:
Even though modules are not subject to same SemVer constraints as the core because they are not supposed to be decorated, this is a little out of the scope of the PR

Two suggestions:
1. Remove this and make this PR "only about PHPStan typing"
2. Keep this but update the PR description 😉 and title

Reply:
Done for 2